app/controllers/lookup_keys_controller.rb

```diff
@@ -4,9 +4,8 @@ class LookupKeysController < ApplicationController
   before_filter :find_resource, :only => [:edit, :update, :destroy], :if => Proc.new { params[:id] }
 
   def index
-    @lookup_keys = resource_base.search_for(params[:search], :order => params[:order])
-                                .includes(:param_classes)
-                                .paginate(:page => params[:page])
+    @lookup_keys = resource_base.search_for(params[:search], :order => params[:order]).includes(:puppetclass)
+                       .paginate(:page => params[:page])
```

Comment:
nitpick: identation looks off? maybe keep the additional methods per line as it was before.